src/collections/_documentation/meta/terms.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Terms of Service'
+sidebar_order: 1
+---
+
+Sentry provides a way to self-serve most legal and compliance documentation for your organization. You'll find these by navigating to _Your Organization's Settings_ > Legal & Compliance.
```

Comment:
Suggest adding: "When you’re logged in to your Sentry account, you can review and accept these agreements as an authorized individual on behalf of your organization."